RELEASE.md

```diff
@@ -0,0 +1,79 @@
+This is an overview of how to run wgpu releases.
+
+## Structure
+
+We do a major breaking release every 12 weeks. This happens no matter the status of various in-flight projects.
+
+We do a patch releases as needed in the weeks between major releases. Once a new major release is cut, we stop doing patch releases for the previous major release unless there is a critical bug or a compilation issue.
```

Comment:
thought: This seems particularly important for users to understand. Not sure if there's anything actionable here, but this was a nice bit of clarity to have.